common/common.go

```diff
@@ -60,7 +71,8 @@ func initEqNameToWeapon() {
 	eqNameToWeapon["scar20"] = EqScar20
 	eqNameToWeapon["sg556"] = EqSG556
 	eqNameToWeapon["smokegrenade"] = EqSmoke
-	eqNameToWeapon["ssg08"] = EqScout
+	eqNameToWeapon["smokegrenadeprojectile"] = EqSmoke
+	eqNameToWeapon["scout"] = EqScout
```

Comment:
Is there a demo out there where this is actually necessary? This name is never used to my knowledge, maybe add it to the TODO above instead about unmapped weapons.

Reply:
It's not something that I really considered. I just happened to stumble upon and added it, I think. I'll remove it again.